contribs/ev/src/main/java/org/matsim/contrib/ev/strategic/StrategicChargingUtils.java

```diff
@@ -287,11 +290,11 @@ static public void configureStanadlone(Config config) {
     }
 
     /**
-     * Sets up the controller
+     * Sets up the controler
```

Comment:
Change back to "controller" to avoid wrong writing. We even encourage people to change the name in the config group now :) 
https://github.com/matsim-org/matsim-libs/blob/506e4627b8c449a7ddf920da67730ffff405dab4/matsim/src/main/java/org/matsim/core/config/ConfigAliases.java#L28